app/controllers/admin/auditor_controller.rb

```diff
@@ -4,7 +4,7 @@
 class Admin::AuditorController < ApplicationController
 
   before_action :require_auditor_role
-  before_action -> { @org = get_org }, olny: [:report, :reports]
+  before_action -> { @org = get_org }, olny: [:report, :reports, :build]
```

Comment:
olny -> only
https://guides.rubyonrails.org/action_controller_overview.html#filters

I'd also refactor the get_org method to set the @org variable, then you could
```
before_action :get_org, only: [:report, :reports, :build]
```
which would read cleaner.